src/pages/Home.vue

```diff
@@ -47,6 +51,7 @@ export default {
     },
 
     async prepareTableData(plioIdList) {
+      this.tableData = this.dummyTableData;
```

Comment:
if we use the pending variable or something else, the dummy table values can be set within `Table.vue` and not Home.vue and then we can simply populate `this.tableData` without creating a new `tableData` var.